www/lang/id-id.json

```diff
@@ -200,7 +200,7 @@
   "save file as": "Simpan berkas sebagai",
   "files": "Files",
   "help": "Help",
-  "support me": "Support me",
+  "support me": "Dukung saya",
   "connecting": "Menghubungkan",
   "file has been deleted": "{file} telah dihapus!",
   "feature not available": "Fitur ini hanya tersedia untuk versi aplikasi berbayar.",
```

Comment:
please change this to 'fitur tidak tersedia'

Reply:
how??

Reply:
edit and commit again